vm/rust/src/jsonrpc.rs

```diff
@@ -58,6 +65,79 @@ struct StateDiff {
     replaced_classes: Vec<ReplacedClass>,
 }
 
+impl StateDiff {
+    pub fn from_state_maps(state_maps: &StateMaps) -> Self {
+        Self {
+            storage_diffs: state_maps
+                .storage
+                .iter()
```

Comment:
This can be `into_iter`, meaning that you are "consuming" the `storage` field in the function. This will also be possible after you take ownership of `state_map` as well, otherwise I think it is not possible